extensions/amp-analytics/0.1/cid-impl.js

```diff
@@ -422,10 +426,12 @@ function getEntropy(win) {
       win.Math.random() + win.screen.width + win.screen.height);
 }
 
+
 /**
- * @param {!Window} window
+ * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampdoc
  * @return {!Cid}
+ * @private visible for testing
  */
-export function installCidService(window) {
-  return fromClass(window, 'cid', Cid);
+export function installCidServiceForDoc(ampdoc) {
```

Comment:
OK, I now understand we don't need this method after compilation. Either change it to `ForTesting` or dropping this method is fine.